mitodl/teachersportal
# Move to django-server-status.

#### What are the relevant tickets?

Refs #203 
#### What's this PR do?

Moves our server status module to the one we released as open source.
#### Where should the reviewer start?

urls.py
#### How should this be manually tested?

Visit `/status/?token=test-token`. You should see postgres info before this change and after this change.
#### Any background context you want to provide?

N/A
#### Screenshots (if appropriate)

N/A
#### What gif best describes this PR or how it makes you feel?

![](http://s.mlkshk-cdn.com/r/17GR3.gif)
